src/state/prices/selectors.js

```diff
@@ -6,6 +6,7 @@ import { parsedBalanceValue } from 'core';
 import { getAssetBalance, getBalance } from 'state/account/selectors';
 
 export const getPrices = state => state.prices;
+export const getHistoricalData = state => state.history;
```

Comment:
Whoops I actually didn't use this and the chunk below it. I'll remove it when I get a second!